src/components/Menu/index.js

```diff
@@ -1,11 +1,14 @@
-import React from "react"
+import React, { useEffect } from "react"
 import { Link } from "react-router-dom"
 import { useI18n } from "providers/I18n"
 
 Menu.propTypes = {}
 
+const userIsLogged = localStorage.getItem("user is logged")
```

Comment:
This should be passed down as a prop from the App component. Reading it in 2 places may cause inconsistencies.
Later on it will be read from AuthContext directly